entrypoint.sh

```diff
@@ -23,6 +24,15 @@ if [ "${1##*.}" != "md" ] ; then
 fi
 BASE_FILE="${1%.*}"
 
+DIFF_FILE=
+if [ ! -z "${INPUT_DIFF_FILE}" ]; then
```

Comment:
Style nit: you can say:

```suggestion
if [ -n "${INPUT_DIFF_FILE}" ]; then
```

(-n for nonempty)